### PR TITLE
CRAYSAT-1843: Update docs of CFS V2-V3 on SAT

### PR DESCRIPTION
**Pull request by annapoorna-s-alt**
# Description
In this script will shown how the cfs version could be added and changed, differences and limitations of using cfs v2 and v3

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by vinanti-p — APPROVED** [review]

File: operations/system_admin_toolkit/usage/Change_CFS_Version.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Change the CFS Version
+
+By default, SAT uses Configuration Framework Service (CFS) version three (`v3`).
+Currently, only CFS version `"v2"` can be used with `sat bootsys` and `sat bootprep`.
+Select the CFS version to use for individual commands with the `--cfs-version`
+option. For more information on this option, refer to the man page for a specific
+command.
```
Comment (haasken-hpe) [inline operations/system_admin_toolkit/usage/Change_CFS_Version.md]: If I understand correctly, currently the version of the CFS API used by `sat bootsys` and `sat bootprep` are hard-coded to v2, so those commands are not affected by the CLI or config-file option described below. I think we could make that a little more clear. I hope we'll be able to fix this in the very near future in CRAYSAT-1896.

Here's my suggestion to make this more clear until we address CRAYSAT-1896.

```suggestion
SAT supports specifying the version of the Configuration Framework Service (CFS)
API used by the `sat status` command. By default the `sat status` command uses
version three (V3) of the CFS API. Currently, the `sat bootsys` and `sat
bootprep` commands can only use CFS V2, but support will be added for CFS V3 in
the future.
```

File: operations/system_admin_toolkit/usage/Change_CFS_Version.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- When using CFS V2, the number of entries returned in a single query is
+  limited by the `default_page_size` parameter, which is 1000.
+- If the system contains more than 1000 components, CFS V2 will return
+  an error as the response size is too large for the system to handle in a
+  single query.
+
+## Note
+
+- Currently, `sat bootprep` could only create CFS V2 configurations.
+- The session check of `sat bootsys` is restricted to use only CFS V2 API.
```
Comment (haasken-hpe) [inline operations/system_admin_toolkit/usage/Change_CFS_Version.md]: I would suggest a more descriptive heading and some minor wording improvements:

```suggestion
## Limitations on CFS V3 Support

As mentioned above, usage of the CFS V3 API is only supported for the `sat
status` command. The following commands use CFS V2:

- `sat bootprep` uses the CFS V2 API to create configurations.
- `sat bootsys` uses the CFS V2 API to check for active CFS sessions.
```

**Review by haasken-hpe — APPROVED** [review]
Looks good! Minor comments, mostly around clearly describing which commands support CFS v3 and which don't yet.

I would also ask you to add a reference to this new file in `operations/system_admin_toolkit/README.md` and `operations/system_admin_toolkit/usage/README.md`.

File: operations/system_admin_toolkit/usage/Change_CFS_Version.md (hunk 1 above)
Comment (annapoorna-s-alt) [reply operations/system_admin_toolkit/usage/Change_CFS_Version.md]: Addressed

File: operations/system_admin_toolkit/usage/Change_CFS_Version.md (hunk 2 above)
Comment (annapoorna-s-alt) [reply operations/system_admin_toolkit/usage/Change_CFS_Version.md]: Addressed

File: operations/system_admin_toolkit/usage/Change_CFS_Version.md (hunk 1 above)
Comment (haasken-hpe) [reply operations/system_admin_toolkit/usage/Change_CFS_Version.md]: Oops, I realize that I omitted a paragraph from my suggestion, so we lost the description of using the CLI option. I've added a commit to this PR which fixes that omission. Sorry about that.

**Review by haasken-hpe — APPROVED** [review]

**Pull request merged**